.pylintrc

```diff
@@ -18,7 +18,8 @@ ignore-patterns=
 init-hook="from pylint.config import find_pylintrc; import os, sys;
  sys.path.append('data_steward');
  sys.path.append('tests');
- sys.path.append(os.path.dirname(find_pylintrc()))"
+ sys.path.append(os.path.dirname(find_pylintrc()));
+ sys.setrecursionlimit(8 * sys.getrecursionlimit())"
```

Comment:
Increasing the recursion depth looks viable- my only suggestion is that we set it to a constant (e.g. 8000) rather than multiply the current limit by some factor. It is possible that the recursion limit gets set by circle (or wherever pylint is being run) prior to this hook; the multiplier approach would be brittle to this whereas a constant would not.

Reply:
Yes, I agree. That would be more resilient.